package.json

```diff
@@ -23,19 +23,15 @@
     "cypress:run:ci": "cypress run",
     "build": "yarn plugin-helpers build",
     "plugin-helpers": "node ../../scripts/plugin_helpers",
-    "postbuild": "echo Renaming build artifact to [$npm_package_config_id-$npm_package_version.zip] && mv build/$npm_package_config_id*.zip build/$npm_package_config_id-$npm_package_version.zip"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
+    "postbuild": "echo Renaming build artifact to [$npm_package_config_id-$npm_package_version.zip] && mv build/$npm_package_config_id*.zip build/$npm_package_config_id-$npm_package_version.zip",
+    "postinstall": "husky install"
```

Comment:
why postinstall?

Reply:
Updated the script

Reply:
> why postinstall

So that when a developer runs bootstrap, they don't need to manually run the `husky install` command. This will automatically take care of that